mage_ai/streaming/sources/google_cloud_pubsub.py

```diff
@@ -136,7 +140,8 @@ def batch_read(self, handler: Callable) -> None:
                 for received_message in response.received_messages:
                     # self._print(f'Received: {received_message.message.data}.')
                     message_values.append(
-                        dict(data=received_message.message.data.decode())
+                        dict(data=received_message.message.data.decode(),
```

Comment:
You can do 
```
dict(data=..., metadata=dict(attributes=...))
```
Here is the format of the message that sink expects: https://github.com/mage-ai/mage-ai/blob/master/mage_ai/streaming/sinks/base.py#L43-L51

Reply:
@wangxiaoyou1993 updated